cmd/dashboard.go

```diff
@@ -76,19 +83,81 @@ var dashboardCmd = &cobra.Command{
 }
 
 func runPortforward(podName string, localPort int) error {
-	cmd, err := exec.LookPath("kubectl")
+	stopChannel := make(chan struct{}, 1)
+	readyChannel := make(chan struct{})
+
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, os.Interrupt)
+	defer signal.Stop(signals)
+
+	go func() {
+		<-signals
+		if stopChannel != nil {
+			close(stopChannel)
+		}
+	}()
+
+	// Open the Dashboard in a browser when the port-forward is established
+	go func() {
+		<-readyChannel
+		openInBrowser(fmt.Sprintf("http://localhost:%d", localPort))
```

Comment:
we now open the dashboard in the browser after the port-forward connection is established, so this fixes #64

Reply:
👍